front/src/pages/Home/Home.tsx

```diff
@@ -26,16 +27,31 @@ const Home = () => {
         <ul className={styles.todosList}>
           {todos.map(todo => (
             <div className={styles.todoWrapper} key={todo.id} onClick={() => handleTodoClick(todo.id, !todo.isDone)}>
-              <span className={styles.checkIcon}>
-                {todo.isDone ? <i>&#9745;</i> : <i>&#9744;</i>}
-              </span>
               <li className={styles.todoItem}>
-                {todo.title}
-                { todo.importance !== 'ordinary'
-                && <div className={`${styles.ribbon} ${styles.ribbonTopRight}`}>
-                  <span style={{backgroundColor: importanceTheme[todo.importance]}}>{todo.importance}</span>
+                <div className={styles.todoItemContent}>
+                  <span className={styles.checkIcon}>
+                    {todo.isDone ? <i>&#9745;</i> : <i>&#9744;</i>}
```

Comment:
we have discussed this before, now I'm sure it's better to move entities like `&#9745;` into separate components. Using them literally looks like magic in bad sense, we cannot even validate we use _correct one_ without starting project/rendering the page

even if component will not get any props: 

```
function CheckedIcon() {
  return <>&#9745;</>
}
```

we will have it named and also it could be possible to _easily_ replace it with ordinary icon/image/etc